pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -15,10 +15,12 @@ stdenv.mkDerivation rec {
     ++ lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
 
   configurePhase = ''
+    runHook preConfigure
     ./configure PREFIX=''${!outputDev} \
```

Comment:
Can we use configureFlags here instead?

Reply:
No, `oconfigure` doesn't like the flags `mkDerivation` always sets which is why used a manual `configurePhase` in the first place.